11.2.основы_работы_со_списками/11.2.6.py

```diff
@@ -0,0 +1,4 @@
+evens = [2, 4, 6, 8, 10, 12, 14, 16, 18, 20]
+average = (sum(evens) / len(evens))
```

Comment:
Скобки тут не нужны :)
```suggestion
average = sum(evens) / len(evens)
```